packages/core/src/com/communication.ts

```diff
@@ -339,6 +339,14 @@ export class Communication {
         return this.environments[envName]?.host;
     }
 
+    public getRegisteredEnvironmentNames(): string[] {
+        return Object.keys(this.environments).filter((id) => id !== '*');
```

Comment:
that's not the names of the environments. it's the instance id. what's the use case to need access to this list?

Reply:
ok, fixed the name of the method.
it is supposed to be used to get all running envs in order to clear them.

here is what I'm going to use this:
```
        const environmentsToDispose = communication.getRegisteredEnvironments();
        environmentsToDispose.forEach((environment) => {
            if (environment !== electronEnv.env) {
                communication.clearEnvironment(environment);
            }
        });
        communication.resetDisposedEnvironments();
```